tests/cypress/support/index.js

```diff
@@ -1,3 +1,5 @@
+import "./commands";
```

Comment:
nit: is there a reason for not importing `steps.js` directly? `commands.js` seems to be redundant.

Reply:
then let's just import `steps` directly. If in the future you need to break into more files, we can then do that, but no need right now